homeassistant/components/picnic/__init__.py

```diff
@@ -36,6 +37,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    # Register the services
+    await async_register_services(hass)
```

Comment:
We should probably unregister the service when unloading the last loaded config entry.

Example:
https://github.com/home-assistant/core/blob/a848dc11556624f8ebf2a09aff7192b84ab4f66e/homeassistant/components/simplisafe/__init__.py#L447-L456